benchmark/main.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+$:.push File.expand_path("lib", __dir__)
```

Comment:
Should this be `File.expand_path("../lib", __dir__)` so it points at the project root directory.